plot_sweep.py

```diff
@@ -20,36 +20,33 @@
 # from collections import defaultdict
 from datetime import datetime as dt
 from torchbenchmark.util.data import load_data_dir, load_data_files
-from torchbenchmark.score.compute_score import compute_score
-from torchbenchmark.score.generate_score_config import generate_bench_cfg
-
+from torchbenchmark.score.compute_score import TorchBenchScore
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description=__doc__)
-    parser.add_argument("data_dir", nargs='+', 
-                        help="One or more directories containing benchmark json files. " 
+    parser.add_argument("data_dir", nargs='+',
```

Comment:
Also sort the file before determining the first json in the first directory. Otherwise the result may not be stable.

Reply:
Here, we enforce the user to provide the reference json. So this check is not required